### ISSUE TITLE
Is it possible to declare one of multiple base types?

### ISSUE THREAD

**Issue by Chaz0r** [GiovanniCardamone/class-schema#7]
Hi and thanks for your lib. I have a small problem and I am not sure if it simply is not possible or if i'm holding something wrong :D

I have a class like this:

```
export class OneFilter {
    value: string | boolean
}
```

I simply want to declare my value in the schema as either string or boolean. Is that possible? The of functions seem only to work with other schema classes.

The result should be something like this:

```
{
  "anyOf": [
    { "type": "string"},
    { "type": "boolean"}
  ]
}
```

Thanks for the help!

**Comment by GiovanniCardamone** [comment]
not at the moment,
i think a nice solution could be something like:

```
useAnyOf(String, Boolean)
```

how do you see that?

**Comment by GiovanniCardamone** [comment]
if you have to use it in a property class you can do something like this:

```
@schema()
class OneFilter {
  @anyOf([String, Number])
  value: string | number
}
```

**Comment by GiovanniCardamone** [comment]
I did checked using decorator `@anyOf` on property and actually give wrong output.

I will fix this soon, but right now it's not available,

also i'll create a utils `useAnyOf(PrimitiveOrSchema1, PrimitiveOrSchema1, ...)` to directly use this not in an object

**Comment by GiovanniCardamone** [comment]
Hi @Chaz0r, I fixed this behavior in 0.25.3

you can upgrade to have a look

Here an example i used in tests

https://github.com/GiovanniCardamone/class-schema/blob/0d3ac30bb9951b1a18003db5c5e2837d5f46ebb8/tests/lib.test.ts#L1382-L1437
